File: 02 Programming Fundamentals with Python May 2022/11 Functions - Exercise/04. Odd and Even Sum.py
def odd_even_sum(number):
    number = str(number)
    sum_even = 0
    sum_odd = 0
    for i in number:
        if int(i) % 2 == 0:
            sum_even += int(i)
        else:
            sum_odd += int(i)
    return f"Odd sum = {sum_odd}, Even sum = {sum_even}"


num = int(input())
print(odd_even_sum(num))

File: 06 JS-Front-End February 2023/05 Functions and Statements - Exercises/04. Odd And Even Sum.js
function printOddEvenSums(number) {
    let numbers = number.toString().split("").map(Number);
    let sumEven = sum(numbers.filter((x) => x % 2 === 0));
    let sumOdd = sum(numbers.filter((x) => x % 2 != 0));
    return `Odd sum = ${sumOdd}, Even sum = ${sumEven}`;

    function sum(array) {
        let sum = 0;
        for (const element of array) {
            sum += element;
        }
        return sum;
    }
}

console.log(printOddEvenSums(1000435));